### DISCUSSION TITLE
Troubles with Remote Share

### DISCUSSION THREAD

**Discussion by MasterMD** [music-assistant/support#2082] [Q&A] [answered]
My HomeAssistant writes its back-up files to my NAS (Synology; SMB version 1 disabled).
When I try to define a music provider with Remote Share all parameters seem to be correct,
but at the moment of Saving it reports: SMB mount error (error 95), operation not supported.
How can I solve this?

**Comment by OzGav** [comment]
Google the error. For example https://www.abrandao.com/2022/01/fix-mount-error95-operation-not-supported-smb/

**Reply by MasterMD** [reply, marked as answer]
OK, thanks!
I didn't know how to fix that for Music Assistant, but I succeeded by adding the version info to the advanced settings:
noserverino,file_mode=0775,dir_mode=0775,uid=0,gid=0,vers=2.0

**Reply by sbaroque** [reply]
Hello, I have a Synology NAS and i can not join the SMB via Music assistant ? with mount error 13, I have adding noserverino,file_mode=0775,dir_mode=0775,uid=0,gid=0,vers=2.0 but its the same error ?? have you a solution for me ? thanks a lot

**Reply by OzGav** [reply]
That is a permission denied error. So check the folder permissions

**Reply by sbaroque** [reply]
I have Check the permissions i have open permission for the user (administrator synology) and when i share with Windows its ok ! ….

**Reply by OzGav** [reply]
You can try different vers values depending on what you are running although I just checked mine and all I have is

`noserverino,file_mode=0775,dir_mode=0775,uid=0,gid=0`

**Reply by sbaroque** [reply]
yes of course , i have this paramaters but its not the solution. I have the same message error : 

<img width="506" height="206" alt="image" src="https://github.com/user-attachments/assets/7fd247c2-0c81-44ef-8035-a3e8712a9e11" />
<img width="506" height="206" alt="Capture d'écran 2025-09-15 231137" src="https://github.com/user-attachments/assets/0c901420-1aa1-4d33-ae5a-c0559d3385d4" />

**Reply by OzGav** [reply]
That’s a different error than you advised before. I’m looking at the` /tmp/` in that message. Have you put that in one of the fields in MA? Possible it should just be `tmp`

**Reply by sbaroque** [reply]
Hello, no i have not put /tmp/ in MA , i must had /tmp/ ? i m lost

**Reply by OzGav** [reply]
Show a screenshot of the file system provider configuration with all of the entries you have made

**Reply by sbaroque** [reply]
Hello, its a screenshot of my entries : 
![Capture d’écran 2025-09-17 200600](https://github.com/user-attachments/assets/3444d6d5-1c75-44b8-93da-f8e04cc88e94)

**Reply by OzGav** [reply]
If you have already tried this
`noserverino,file_mode=0775,dir_mode=0775,uid=0,gid=0`

then try 
`noserverino,file_mode=0775,dir_mode=0775,uid=0,gid=0,vers=3.0`

and then
`noserverino,file_mode=0775,dir_mode=0775,uid=0,gid=0,vers=1.0`

and then
`noserverino,file_mode=0775,dir_mode=0775,uid=0,gid=0,sec=ntlm`
 
and then
`noserverino,file_mode=0775,dir_mode=0775,uid=0,gid=0,sec=none`

There appears to be some configuration difference with your NAS to the standard.

**Reply by sbaroque** [reply]
Hello, I have try all this verions of configurations 
i have différent message like this one : 
![Uploading Capture d’écran 2025-09-18 192556.jpg…]()

**Reply by sbaroque** [reply]
Hello, I have try all this verions of configurations 
i have différent message like this one : 
![Capture d’écran 2025-09-18 192556](https://github.com/user-attachments/assets/e8d3c601-a17a-4ea1-9bd1-788b4b8ae8ee)

**Reply by OzGav** [reply]
That shows it is connecting but PERMISSIONS are a problem. Create a dedciated user for MA to use. Give it read and write just to the ziks_rock folder

**Reply by sbaroque** [reply]
Yes ok i have create a new user musicassistant but its the same message error ;... 
![Capture d’écran 2025-09-19 221327](https://github.com/user-attachments/assets/c998ddfc-fc10-4f55-acb1-0699da361611)

**Reply by OzGav** [reply]
No this a different error now. I’m afraid I can’t help much more. This is not a MA problem. Look at how you have installed MA, do some more searches on the errors and maybe ask on dedicated Linux networking forum.

**Reply by sbaroque** [reply]
i m afraid !! ouahh !! beautiful reaction for a helper ... SOS and go out for me ! thank you very much ... there is no one for me ? please help ! or i delete MA music assistant
